apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts

```diff
@@ -47,17 +47,7 @@ export class GetMessages {
       }
     }
 
-    const totalCount = await this.messageRepository.count(
-      query,
-      COUNT_LIMIT
-      /*
-       * todo NV-2161 in version 0.16:
-       *  update option as below,
-       *  update below:  hasMore = feed.length < totalCount
-       *  remove totalCount
-       * { skip: command.page * LIMIT, limit: LIMIT + 1 }
-       */
-    );
+    const totalCount = await this.messageRepository.count(query, COUNT_LIMIT);
```

Comment:
I wonder if we should remove the count limit here because it is not used by the notification-center